packages/steppp-react/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-    "extends": "../../tsconfig.json",
-    "compilerOptions": {
-        "outDir": "dist",
-        "jsx": "react-jsx"
-    },
-    "include": [
-        "./src"
-    ]
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "jsx": "react-jsx",
+    "emitDeclarationOnly": false
```

Comment:
This causes `tsc` to spit the compiled code into the `dist` directory as well -- not only the `.d.ts` files.